src/redis_lock/__init__.py

```diff
@@ -156,7 +157,9 @@ def __init__(self, redis_client, name, expire=None, id=None, auto_renewal=False)
         self._held = id is not None
         self._name = 'lock:'+name
         self._signal = 'lock-signal:'+name
-        self._lock_renewal_interval = expire*2/3 if auto_renewal else None
+        self._lock_renewal_interval = (float(expire)*2/3
+                                       if auto_renewal
+                                       else None)
```

Comment:
You decide. I think, there is also should be a test for that change, don't merge the PR plz.